api/data/pipeline.py

```diff
@@ -6,13 +6,14 @@
 from .collection import store_database_for_eys_gene
```

Comment:
this file is no longer supported, for testing use `tests` folder and pipeline.ipynb inside

Reply:
Though you still can use it as reference, it will be deleted soon

Reply:
Unresolved as pipeline.py still contain changes